src/main/java/mate/jdbc/service/impl/DriverServiceImpl.java

```diff
@@ -0,0 +1,39 @@
+package mate.jdbc.service.impl;
+
+import java.util.List;
+import mate.jdbc.dao.DriverDao;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Driver;
+import mate.jdbc.service.DriverService;
+
+@Service
+public class DriverServiceImpl implements DriverService {
+    @Inject
+    private DriverDao driverDao;
+    
+    @Override
+    public Driver create(Driver driver) {
+        return driverDao.create(driver);
+    }
+    
+    @Override
+    public Driver get(Long id) {
+        return driverDao.get(id).get();

```

Comment:
Please use .orElseThrow() construction to rethrow exception with meaningful message in case driver wasn't found